products/BellHybrid/apps/application-bell-main/include/application-bell-main/ApplicationBellMain.hpp

```diff
@@ -26,7 +26,7 @@ namespace app
                                      std::string parent                  = "",
                                      StatusIndicators statusIndicators   = StatusIndicators{},
                                      StartInBackground startInBackground = {false},
-                                     std::uint32_t stackDepth            = 10240);
+                                     std::uint32_t stackDepth            = 1024 * 15);
```

Comment:
it's not equal, again - we were below or something happened?

Reply:
Updated FreeRTOS sources captured stack overflow here ;)

Reply:
That's a lot :o :+1: